LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Joana Be.
+Copyright (c) 2018 Tobias Markus, Daniel-Amadeus Johannes Glöckner
```

Comment:
we should probably use "snailsnap" als the copyright holder here, like we've done in the mosaic repo, but that's up to you if you want to change it.

Reply:
I'm not sure whether that's a good idea considering there is no legal entity behind snailsnap, so assigning copyright to snailsnap isn't possible IIUC.